File: onebot-sdk/src/main/java/cn/evole/onebot/sdk/event/request/FriendAddRequestEvent.java
package cn.evole.onebot.sdk.event.request;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.experimental.SuperBuilder;

/**
 * Created on 2022/7/8.
 *
 * @author cnlimiter
 */
@Data
@AllArgsConstructor
@SuperBuilder(toBuilder = true)
@EqualsAndHashCode(callSuper = true)
public class FriendAddRequestEvent extends RequestEvent {
}

File: onebot/src/main/kotlin/sdk/event/request/FriendAddRequestEvent.kt
package cn.evolvefield.onebot.sdk.event.request

import lombok.AllArgsConstructor
import lombok.Data
import lombok.EqualsAndHashCode
import lombok.experimental.SuperBuilder

@Data
@AllArgsConstructor
@SuperBuilder(toBuilder = true)
@EqualsAndHashCode(callSuper = true)
class FriendAddRequestEvent : RequestEvent()
